src/figuresClasses.ts

```diff
@@ -1,11 +1,77 @@
-export interface Figure {}
+export interface Figure {
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape: string = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
```

Comment:
[CODE KNOWLEDGE] - You don't need to assign the constructor parameters to properties manually if you are using public in the parameter list. TypeScript does this automatically.